braph2genesis/graphs/_MultilayerBUT.gen.m

```diff
@@ -1,10 +1,10 @@
 %% ¡header!
-MultilayerBUT < MultilayerWU (g, multilayer binary undirected with fixed threaholds graph) is a multilayer binary undirected with fixed threaholds graph.
+MultilayerBUT < MultilayerWU (g, multilayer binary undirected with fixed thresholds graph) is a multilayer binary undirected with fixed thresholds graph.
 
 %%% ¡description!
-In a multilayer binary undirected with fixed thresholds (BUT) graph, layers 
-consist of binary undirected (BU) multilayer graphs derived from the same
-weighted supra-connectivity matrices binarized at different thresholds. Layers 
+In a multilayer binary undirected with fixed thresholds (BUT) graph, the layers are those of binary undirected (BU)
+multiplex graphs derived from the same weighted supra-adjacency matrix binarized at different thresholds. 
```

Comment:
**multilayer** instead of multiplex, sorry I introduced this error before I believe.